File: Iniciante/PHP/exercise_1044.php
<?php

$valores = readline();

$valores = explode(" ", $valores);

$valores[0] = intval($valores[0]);
$valores[1] = intval($valores[1]);

sort($valores);

$valorK = $valores[1]/$valores[0];

if (($valorK - floor($valorK)) === 0.0) {
    print_r("Sao Multiplos\n");
} else {
    print_r("Nao sao Multiplos\n");
}

//Concluído!

File: Iniciante/Python/exercises from 1000 to 1099/exercise_1044.py
import math

valores = [int(x) for x in input().split()]
valores.sort()
valorA, valorB = valores

valorK = valorB/valorA
isMultiple = (valorK - math.floor(valorK)) == 0.0

if isMultiple:
    print('Sao Multiplos')
else:
    print('Nao sao Multiplos')